cassovary-core/src/test/scala/com/twitter/cassovary/graph/MemoryMappedDirectedGraphSpec.scala

```diff
@@ -72,10 +60,9 @@ class MemoryMappedDirectedGraphSpec extends WordSpec with Matchers {
       graph1.getNodeById(-1) should be(None)
       graph1.getNodeById(6) should be(None)
       graph1.getNodeById(1 << 29) should be(None)
-      tempFile.deleteOnExit()
```

Comment:
Ack, I think I removed that during debugging and forgot to restore it. I'll restore it.